pkg/projectfile/projectfile.go

```diff
@@ -1005,7 +1005,7 @@ func createCustom(params *CreateParams, lang language.Language) (*Project, error
 	}
 
 	content := params.Content
-	if content == "" && lang.String() != "" {
+	if content == "" && lang != language.Unset && lang != language.Unknown {
```

Comment:
Looks good to me! IIRC my issue with the last iteration was mainly that we were only checking for one or the other.